### PR TITLE
NSI-3133 ECO backscattering

### PR DESCRIPTION
**Pull request by SBS-randerson**
# Description

Adds ECO beta for backscattering

Closes: NSI-3133

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Select the type of tests you added for these changes

- [ ] Unit
- [ ] Integration
- [ ] Component
- [ ] E2E
- [x] User

## Did you test these changes on all platforms (if applicable to the JIRA item?)

- [x] Windows
- [ ] MacOS

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have cleared any run data/output from the ipynb file(s)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] My code is up to date with the branch I'm requesting to merge into

### CODE REVIEW

File: src/sbs/process/conversion.py [hunk 1]
```diff
@@ -902,6 +903,25 @@ def convert_ECO_turbidity_val(
...
+):
+    """Converts a raw value for backscatter channel on a ECO-FLNTU to β (1/m/sr) 
+
+    Args:
+        rawBeta(float): raw counts for digital, raw volts for analog
+        ScaleFactor (float): β/count for digital, β/V for analog
+        Vblank (float): dark counts for digital, V for analog
+
+    Returns:
+        float: converted beta in β (1/m/sr)
```
Comment (SBS-jtemplin) [inline src/sbs/process/conversion.py:919]: The units are sort of confusing and make it look equivalent to `sr/m`. How about `1/(m*sr)` or `1*m^-1*sr^-1`. The latter's a bit cumbersome without better formatting though.

Sorry for the slight delay in getting to this btw

**Comment by SBS-randerson** [comment]
I'm going to hold off on this PR since there is going to be a rework of how we're handling ECO

File: src/sbs/process/conversion.py (hunk 1 above)
Comment (SBS-EREHM) [reply src/sbs/process/conversion.py:919]: I suggest `m-1 sr-1` or `1/m/sr`

**Comment by SBS-randerson** [comment]
Won't merge in, will create a generic ECO function in the future.

**Pull request closed without merging**